### PR TITLE
The News Week home page clone merge request

### PR DESCRIPTION
**Pull request by temesghentekeste**
I created a branch called "homepage" and developed that branch.

- On the initial commit, I created a basic structure for the page and added CDN links to Bootstrap, JQuery, Fontawesome, and Google Fonts.

 - "Mobile first" approach is used in order to achieve responsiveness principles.

 - Elements are positioned using Bootstrap and a custom CSS style.

Please review my project before I merge my branch.

Thanks in advance!

### CODE REVIEW

**Comment by santiago-dev96** [comment]
## Pull Request under review by @santiago-rodrig (TSE)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+
+## Mobile
+
+![screenshot](https://github.com/temesghentekeste/newsweek-clone/blob/homepage/img/app_screenshot_mobile.png)
+
+## Built With
+
+- HTML,
+- CSS,
+- VS Code
```
Comment (santiago-dev96) [inline README.md]: The text editor is not important

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+- Firefox Developer Edition dev tools
```
Comment (santiago-dev96) [inline README.md]: Neither the browser tools

File: css/custom.css [hunk 3]
```diff
@@ -0,0 +1,637 @@
+:root {
+  --primary-color: #f72210;
+  --secondary-color: #fff;
+  --black-color: #000;
+  --bg-color: #eee;
+  --primary-font: 'Roboto Condensed', sans-serif;
+  --secondary-font: Arial, Helvetica, sans-serif;
+  --tirtiary-font: 'Arimo', sans-serif;
+  --font-summary-heading: 1.0625rem;
+}
```
Comment (santiago-dev96) [inline css/custom.css:10]: Great use of CSS variables!

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    let position = $(this).scrollTop();
+    let $navbarToggler = $('.navbar-toggler');
+    let $logoContainer = $('.main-logo-container');
+    let $mainLogo = $('.main-logo');
+    let $weatherContainer = $('.weather');
+    let $date = $('.date');
+
+    console.log(position);
+    console.log($navbarToggler)
+    console.log($logoContainer)
```
Comment (santiago-dev96) [inline js/scripts.js]: Don't push console logs

File: js/scripts.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    //   $('.red-nav').removeClass('top-nav-onScroll')
+    //   $('.top-nav-container').addClass('container');
+    //   $navbarToggler.css({'display': 'none'});
+    //   $logoContainer.css({'order': '0', 'height': '90px','width':'324px', 'margin-right': 'inherit'});
+    //   $mainLogo.css({'height': '65px', 'margin-top':'1rem'});
+    //   $weatherContainer.css({'margin-right': 'inherit'});
+    //   // $mainLogo.css({'height': '60px', 'padding-bottom': '3rem'})
+    //   // $('.red-nav').css({'height': '95px'})
+
+    // }
```
Comment (santiago-dev96) [inline js/scripts.js]: Don't push commented code

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required! :no_entry_sign:

On small mobiles (~360 widths), the sign-in link doesn't display properly.

image

Please go ahead and implement the changes I requested. We are no assessing JavaScript for HTML & CSS projects, keep that in mind in the future. Have a nice day :wave:!

You can find me in slack as **srodrig**.

**Comment by temesghentekeste** [comment]
Thanks, @santiago-rodrig for your review and I have fixed them all as per your suggestions.

**Review by sumancrest0001 — APPROVED** [review]
**Congratulation @temesghentekeste :clap:  :clap:** 
You have done a terrific job and it looks similar to the original site. It's a YES from my side. You can merge the PR and submit the project completion form. 
[OPTIONAL] Try to avoid !important in your stylesheet. It is not a good practice. 

**HAPPY CODING !!!**
mission

**Comment by temesghentekeste** [comment]
Thanks, @sumancrest0001 for approving the project and I will try to apply your recommendation in my next project.

**Pull request merged**